source/develop/release-management/features/network/networkfilterparameters.html.md

```diff
@@ -226,6 +230,14 @@ The feature depends on [Network Filter for vNIC profiles][2], which is already a
 
 [9]: https://libvirt.org/formatnwfilter.html#nwfelemsReservedVars
 
+[NicNetworkFilterParameters in oVirt Engine's REST-API][10]
+
+[10]: http://ovirt.github.io/ovirt-engine-api-model/4.2/#services/nic_network_filter_parameter
+
+[Example of using Network Filter Parameters via the REST-API][11]
+
+[11]: https://gerrit.ovirt.org/#/c/78045/13/basic-suite-master/test-scenarios/004_basic_sanity.py
```

Comment:
Maybe a link to the latest version of the test instead of the patch would be better:
https://github.com/oVirt/ovirt-system-tests/blob/master/basic-suite-master/test-scenarios/004_basic_sanity.py#L412